server/db/index.js

```diff
@@ -1,35 +1,64 @@
 const db = require('./db')
-const { User, Snippet, Stretch, Comment } = require('./models')
+const models = require('./models')
+const {
+  User,
+  StretchAnswer,
+  Stretch,
+  Comment,
+  Cohort,
+  CohortStretch,
+  Category,
+  CohortUser
+} = models
 
 function initDb(force = false) {
   return db.authenticate().then(() => {
     // Sequelize associations
     // --------------------------
-    // A user has many snippets
-    Snippet.belongsTo(User)
-    User.hasMany(Snippet)
 
-    // A stretch has many snippets
-    Snippet.belongsTo(Stretch)
-    Stretch.hasMany(Snippet)
+    // Many-to-Many between User and Cohort
+    CohortUser.belongsTo(Cohort)
+    Cohort.hasMany(CohortUser)
+    CohortUser.belongsTo(User)
+    User.hasMany(CohortUser)
 
-    // A user and snippet both have many comments
+    // Comment belongs to User
     Comment.belongsTo(User)
     User.hasMany(Comment)
 
-    Comment.belongsTo(Snippet)
-    Snippet.hasMany(Comment)
+    // StretchAnswer belongs to User
+    StretchAnswer.belongsTo(User)
+    User.hasMany(StretchAnswer)
+
+    // Stretch belongs to User
+    Stretch.belongsTo(User, { as: 'author' })
```

Comment:
Just want to clarify what this means. It means that when a stretch is created by a User with isAdmin:true, that stretch has an authorId which is really the userId of the user who created the stretch. 

If so, I'm imagining most of the create stretch logic as a class level method on the user model, and some kind of validation to check a user's isAdmin before allowing them to create the stretch.